### DISCUSSION TITLE
AsusFX504 - 0.7.0

### DISCUSSION THREAD

**Discussion by RobyRew** [RobyRew/ASUS-FX504GE-Hackintosh_OpenCore#4] [General]
### News / Fixes:
- Fixed Backlight issue after boot with `-igfxblr` boot argument.
- Jack 3.5mm its now working correctly.
- Updated all kexts and boot files to latest versions.
- Updated the config.plist.
- Fixed OpenCanopy issues

### Old Know Issues:
- HDMI have issues with some resolutions.

<hr /><em>This discussion was created from the release <a href='https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_Updated_OpenCore_0.6.9/releases/tag/4.0'>AsusFX504 - 0.7.0</a>.</em>

**Comment by wojko6** [comment]
Thanks. Amazing job!
